src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/filters/suppresswithnearbytextfilter/Example3.java

```diff
@@ -9,7 +9,10 @@
   </module>
 </module>
 */
-
+package com.puppycrawl.tools.checkstyle.filters.suppresswithnearbytextfilter;
 // xdoc section -- start
-export FOO=BAR # ok, because violation message is matching suppress pattern
+public class Example3 {
+  // ok, because violation message is matching suppress pattern
+  int a_really_long_variable_name = 10;
```

Comment:
Changed the example from what looks like a shell script (`export foo=bar`) to a Java class with long variable name. imo easier to understand.